Mathlib/AlgebraicTopology/SimplexCategory.lean

```diff
@@ -239,6 +265,14 @@ def mkOfLeComp {n} (i j k : Fin (n + 1)) (h₁ : i ≤ j) (h₂ : j ≤ k) :
       | 0, 2, _ => Fin.le_trans h₁ h₂
   }
 
+/-- The "inert" morphism associated to a subinterval `j ≤ i ≤ k` of `Fin (n + 1)`.-/
+def subinterval {n} (j l : ℕ) (hjl : j + l < n + 1) :
```

Comment:
We may consider the `<` is a (necessary) implementation detail of `Fin`, but using `≤` is probably more convenient.
```suggestion
/-- The "inert" morphism associated to a subinterval `j ≤ i ≤ j + k` of `Fin (n + 1)`.-/
def subinterval {n} (j l : ℕ) (hjl : j + l ≤ n) :
```